packages/cli/src/zosfiles/edit/Edit.utils.ts

```diff
@@ -0,0 +1,328 @@
+     * @param {boolean} useStash - should be true if don't want to overwrite local file when refreshing etag
+     * @returns {LocalFile}
+     */
+    public static async localDownload(session: AbstractSession, lfFile: LocalFile, useStash: boolean): Promise<LocalFile>{
+        // account for both useStash|!useStash and uss|ds when downloading
+        const tempPath = useStash ? path.join(tmpdir(), "toDelete.txt") : lfFile.tempPath;
+        const args: [AbstractSession, string, IDownloadOptions] = [
+            session,
+            lfFile.fileName,
+            {
+                returnEtag: true,
+                file: tempPath
+            }
+        ];
+        if(lfFile.fileType === 'uss'){
+            lfFile.zosResp = await Download.ussFile(...args);
+        }
+        lfFile.zosResp = await Download.dataSet(...args);
+        // TODO: get rid of the fake temp file generated by the useStash case (wouldnt have to do this if the option {overwrite: false} worked)
+        if (useStash){
+            this.destroyTempFile(path.join(tmpdir(), "toDelete.txt"));
+        }
+        return lfFile;
+    }
+
+    /**
+     * Performs appropriate file comparision (either in browser or as a terminal diff) between lf-USS or lf-DS.
+     * Local file (lf) will then be opened in default editor
+     * @param {AbstractSession} session - the session object generated from the connected profile
+     * @param {IHandlerParameters} commandParameters - parameters supplied by args
+     * @returns {Promise<IZosFilesResponse>} - the response generated by {@link CompareBaseHelper.getResponse}
+     * @memberof EditUtilities
+     */
+    public static async fileComparison(session: AbstractSession, commandParameters: IHandlerParameters): Promise<IZosFilesResponse>{
+        const handlerDs = new LocalfileDatasetHandler;
+        const handlerUss = new LocalfileUssHandler;
+        const helper = new CompareBaseHelper(commandParameters);
+        const gui = ProcessUtils.isGuiAvailable();
+        const options: IDiffOptions = {
+            name1: "local file",
+            name2: "mainframe file"
+        };
+
+        helper.browserView = (gui === GuiResult.GUI_AVAILABLE);
+
+        const lf = await handlerDs.getFile1(session, commandParameters.arguments, helper);
+        let mfds: string | Buffer;
+        if (commandParameters.positionals.includes('ds')){
+            mfds = await handlerDs.getFile2(session, commandParameters.arguments, helper);
+        }else{
+            mfds = await handlerUss.getFile2(session, commandParameters.arguments, helper);
+        }
+
+        //if browserview, open diff in browser, otherwise print diff in terminal
+        const response = await helper.getResponse(helper.prepareContent(lf), helper.prepareContent(mfds), options);
+        if (!helper.browserView){
+            console.log('\n'+response.commandResponse);
```

Comment:
The linter is complaining here because we don't allow `console.log` in our codebase. I think this could be refactored to move the logging to console into the command handler and use `params.response.console.log` on the IHandlerParameters API.